http/auth_service.go

```diff
@@ -336,7 +336,7 @@ func (h *AuthorizationHandler) handleGetAuthorizations(w http.ResponseWriter, r
 	ctx := r.Context()
 	req, err := decodeGetAuthorizationsRequest(ctx, r)
 	if err != nil {
-		h.Logger.Info("failed to decode request", zap.String("handler", "getAuthorizations"), zap.Error(err))
+		h.log.Info("failed to decode request", zap.String("handler", "getAuthorizations"), zap.Error(err))
```

Comment:
`Failed`.

Reply:
I used Golang's cmd-shift-R and `log(ger)?\.(Info|Warn|Debug|Error)f?\("[a-z]`.